usaspending_api/api_contracts/contracts/v2/disaster/cfda/loans.md

```diff
@@ -91,8 +91,10 @@ Returns loan spending details of CFDA Programs receiving supplemental funding bu
         + `asc`
 + `sort` (optional, enum[string])
     Optional parameter indicating what value results should be sorted by
-    + Default: `description`
+    + Default: `id`
```

Comment:
I get that this change is to make sure that all API Contracts match and that this will pass Dredd due to changes below, but if someone were to try and actually use all of these sorts the functionality would be off. If we keep the current functionality and add this to the API Contract it might be good to add some sort of disclaimer to the API Contract.

Reply:
I really should read the whole PR and then go back and comment. Just saw the message below.

Reply:
I'm not sure how to clearly communicate in the Contract without adding confusion. There is a new message in the API response, which should help.

This?
```
When `award_type_codes` are provided and `spending_type` = "award" then sorting by `id` and `code` are not perfectly implemented and will instead sort by `description`
```

Reply:
Can just ignore my comment. Realized below you added a message to the response.